config/locales/doorkeeper.zh-CN.yml

```diff
@@ -124,4 +123,9 @@ zh-CN:
           home: '首页'
       application:
         title: 'OAuth 认证'
+     scopes:
```

Comment:
There seems to be some sort of YML syntax error in this file, but I am not sure which exactly.

`I18n::InvalidLocaleData: can not load translations from /home/travis/build/tootsuite/mastodon/config/locales/doorkeeper.zh-CN.yml: #<Psych::SyntaxError: (/home/travis/build/tootsuite/mastodon/config/locales/doorkeeper.zh-CN.yml): did not find expected key while parsing a block mapping at line 19 column 5>`